zulipterminal/ui_tools/boxes.py

```diff
@@ -1566,12 +1565,19 @@ def main_view(self) -> List[Any]:
         any_differences = any(different.values())
 
         if any_differences:  # Construct content_header, if needed
-            TextType = Dict[str, urwid_MarkupTuple]
-            text_keys = ("author", "star", "time", "status")
-            text: TextType = {key: (None, " ") for key in text_keys}
-
-            if any(different[key] for key in ("recipients", "author", "24h")):
-                text["author"] = ("name", message["this"]["author"])
+            TextType = Dict[str, List[urwid_MarkupTuple]]
```

Comment:
This change, with accompanying adjustments, is a perfectly fine separate refactor commit.

Generally more commits are better than fewer, since they're more difficult to separate later. However, note that these are ideally structured commits rather than commits that one does from just adding fixes to previous code in a branch.

Reply:
So this wasn't purely a refactor since adding a new style class to the urwid Text widget needed me to change the TextType also so that it would pass all `make check` checks. 

But I understand what you meant by keeping commits structured, hopefully the newer ones I made are more suitable